chia/cmds/coin_funcs.py

```diff
@@ -71,12 +78,18 @@ async def async_list(args: Dict[str, Any], wallet_client: WalletRpcClient, finge
             [(coin, "") for coin in unconfirmed_additions],
             mojo_per_unit,
             addr_prefix,
+            coin_type,
             paginate,
         )
 
 
 def print_coins(
-    target_string: str, coins: List[Tuple[Coin, str]], mojo_per_unit: int, addr_prefix: str, paginate: bool
+    target_string: str,
+    coins: List[Tuple[Coin, str]],
+    mojo_per_unit: int,
+    addr_prefix: str,
+    coin_type: str,
```

Comment:
This seems more like the ticker or name than the 'type'?

Reply:
changed variable name